src/logic/currentSession/store/reducer/currentSession.ts

```diff
@@ -6,13 +6,16 @@ import { saveCurrentSessionToStorage } from 'src/logic/currentSession/utils'
 import { AppReduxState } from 'src/store'
 
 export const CURRENT_SESSION_REDUCER_ID = 'currentSession'
+const maxViewedSafes = 10
```

Comment:
```suggestion
const MAX_VIEWED_SAFES = 10
```

let's keep constant names consistent